packages/flutter/lib/src/material/scrollbar.dart

```diff
@@ -119,28 +118,23 @@ class _ScrollbarState extends State<Scrollbar> {
   void _updateState(ScrollableState scrollable) {
     if (scrollable.scrollBehavior is! ExtentScrollBehavior)
       return;
-    final ExtentScrollBehavior scrollBehavior = scrollable.scrollBehavior;
-    _scrollOffset = scrollable.scrollOffset;
-    _scrollDirection = scrollable.config.scrollDirection;
-    _contentExtent = scrollBehavior.contentExtent;
-    _containerExtent = scrollBehavior.containerExtent;
+    setState(() {
```

Comment:
There was no setState() here, since it's called by onScrollStarted(), to avoid rebuilding the scrollable in response to a simple tap. See https://github.com/flutter/flutter/pull/5038